recipes/nvjitlink/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set data = load_file_data("pyproject.toml") %}
+{% set name = "nvjitlink" %}
+{% set version = "0.1.0.dev1" %}
+
+package:
+  name: nvjitlink
+  version: "0.1.0.dev1"
+
+source:
+  path: https://github.com/rapidsai/nvjitlink/archive/refs/tags/0.1.0.tar.gz
+  sha256: cb3d502cd183ed173c2202d2d8ea12d918beb57aa671693cbd5489a7b01b33ac
+
+build:
+  noarch: python
+  script:
+    # This is a workaround for library search path issues with the current
+    # cuda-nvcc package. They should be fixed by
+    # https://github.com/conda-forge/cuda-nvcc-feedstock/pull/24, at which
+    # point we can remove this.
+    - export CMAKE_PREFIX_PATH=${PREFIX}/targets/x86_64-linux/lib:$CMAKE_PREFIX_PATH
+    - {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}
+    - cmake >=3.26.4
+    - sysroot_{{ target_platform }} 2.17
+    - cuda-version
```

Comment:
This is added by `{{ compiler('cuda') }}` on CUDA 12. So we can rely on that here

```suggestion
```